src/bin/dual-iir.rs

```diff
@@ -68,7 +67,7 @@ const BATCH_SIZE: usize = 8;
 // 128, there is 1.28uS per sample, corresponding to a sampling frequency of 781.25 KHz.
 const SAMPLE_TICKS_LOG2: u8 = 7;
 
-#[derive(Clone, Copy, Debug, Deserialize, Miniconf)]
```

Comment:
Just a note: I was about to ask why we don't eagerly derive things. But then I looked closer and the serde traits are actually not "common traits" [Types eagerly implement common traits (C-COMMON-TRAITS)](https://rust-lang.github.io/api-guidelines/interoperability.html#types-eagerly-implement-common-traits-c-common-traits)